platform/os_windows.go

```diff
@@ -118,3 +137,48 @@ func KillProcessByName(processName string) {
 	cmd := fmt.Sprintf("taskkill /IM %v /F", processName)
 	ExecuteCommand(cmd)
 }
+
+// executePowershellCommand executes powershell command
+func executePowershellCommand(command string) (string, error) {
+	ps, err := exec.LookPath("powershell.exe")
+	if err != nil {
+		return "", fmt.Errorf("Failed to find powershell executable")
+	}
+
+	cmd := exec.Command(ps, command)
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	cmd.Run()
+
+	return strings.TrimSpace(stdout.String()), nil
+}
+
+// SetSdnRemoteArpMacAddress sets the regkey for SDNRemoteArpMacAddress needed for multitenancy
+func SetSdnRemoteArpMacAddress() error {
+	if sdnRemoteArpMacAddressSet == false {
+		result, err := executePowershellCommand(GetSdnRemoteArpMacAddressCommand)
+		if err != nil {
+			return err
+		}
+
+		// Set the reg key if not already set or has incorrect value
+		if result == "" || result != SDNRemoteArpMacAddress {
```

Comment:
if result != SDNRemoteArpMacAddress means result == "" too so no need for OR here

Reply:
That's not true. result can be some incorrect value "12-12-12-12". We need this OR to check if the result is empty which mean the key is not set or to check if the result is incorrectly set.

Reply:
I meant we should remove the empty check and keep the incorrect value check

Reply:
Thanks addressed this in the latest update